plugins/@grouparoo/iterable/__tests__/export/export-profile.ts

```diff
@@ -182,7 +183,15 @@ describe("iterable/exportProfile", () => {
 
     await runExport({
       oldProfileProperties: {},
-      newProfileProperties: { email, name },
+      newProfileProperties: {
+        email,
+        name,
+        defaultAddress: {
+          street: street1,
+          number: number1,
+          zipcode: zipcode,
+        },
```

Comment:
I don't think that it's likely Grouparoo would provide an object to the plugin like this

Reply:
The `.` way of defining properties seems more likely

Reply:
This is just to add the object there for the first time. But I agree we should use dot notation even the first time.